src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_COEFFICIENT = 2;
+    private Node<K, V>[] table;
+    private float threshold;
+    private int size;
+
+    protected MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = table.length * DEFAULT_LOAD_FACTOR;
```

Comment:
при ініціалізації ми знаємо яка довжина масиву у нас
```suggestion
        threshold = DEFAULT_INITIAL_CAPACITY * DEFAULT_LOAD_FACTOR;
```